vignettes/vroom.Rmd

```diff
@@ -112,6 +112,12 @@ file <- vroom_example("mtcars.csv.gz")
 vroom(file)
 ```
 
+About how it work `vroom()`. After the works of this function (indexing and writes) the already uncompressed file is going to compressed again but into a temporal file (in a `tempdir()`).
```

Comment:
```suggestion
After the `vroom()` function indexes and writes the initial file, the already uncompressed file is going to compressed again but into a temporary file (in a `tempdir()`).
```

Reply:
I think this might be missing an initial pass of decompression (cc @jimhester ?)